src/components/Auth/ForgotPassword.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faEnvelope } from "@fortawesome/free-regular-svg-icons"
+import { faEye } from "@fortawesome/free-regular-svg-icons"
+import Alert, { NotificationType } from "../Alert/Alert"
+
+import { forgotPasswordSubmit } from "./utils"
+import "./Register.scss"
+
+const ForgotPassword: React.FC = () => {
+  const [formValues, setFormValues] = useState({
+    username: "",
+    password: "",
+    code: "",
+  })
+  const [displayError, setDisplayError] = useState({
+    isError: false,
+    message: "",
+  })
+
+  const updateForm = (event: any) => {
+    setFormValues({
+      ...formValues,
+      [event.target.name]: event.target.value,
+    })
+  }
+
+  const submitNewPassword = () => {
+    setDisplayError({
+      ...displayError,
+      isError: false,
+    })
+    forgotPasswordSubmit(
+      formValues.username,
+      formValues.code,
+      formValues.password
+    ).then(message => {
+      setDisplayError({
+        isError: true,
+        message: message as string,
+      })
+    })
+  }
+
+  const signUpForm = (
+    <div className="card">
+      <div className="card-content">
+        <div className="content">
+          <div className="form">
+            <Alert
+              message={"We're sent you a confirmation code, check your email"}
+              notificationType={NotificationType.isSuccess}
+            />
```

Comment:
I suspect this duplicate `Alert` component is causing the issue

Reply:
So when you press forgot password on the signin screen the system sends the user a verification code via email. That's what the alert is for. They need to know that they need that code to reset the password